src/pages/ReportParticipantsPage.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import _ from 'underscore';
+import {
+    View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import Str from 'expensify-common/lib/str';
+import styles from '../styles/styles';
+import ONYXKEYS from '../ONYXKEYS';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import Navigation from '../libs/Navigation/Navigation';
+import ScreenWrapper from '../components/ScreenWrapper';
+import OptionsList from '../components/OptionsList';
+import ROUTES from '../ROUTES';
+import personalDetailsPropType from './personalDetailsPropType';
+
+const propTypes = {
+    /* Onyx Props */
+    // The personal details of the person who is logged in
+    personalDetails: personalDetailsPropType.isRequired,
+
+    // The active report
+    report: PropTypes.shape({
+        // The list of icons
+        icons: PropTypes.arrayOf(PropTypes.string),
+
+        // The report name
+        reportName: PropTypes.string,
+
+        // Array of participants
+        participants: PropTypes.arrayOf(PropTypes.string),
+    }).isRequired,
+
+    // Route params
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            // Report ID passed via route r/:reportID/participants
+            reportID: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+/**
+ * Returns all the participants in the active report
+ *
+ * @param {Object} report The active report object
+ * @param {Object} personalDetails The personal details of the users
+ * @return {Array}
+ */
+const getAllParticipants = (report, personalDetails) => {
+    const {participants} = report;
+
+    return _.map(participants, (login) => {
+        const userPersonalDetail = personalDetails[login];
+        const userLogin = Str.removeSMSDomain(login);
+
+        return ({
+            alternateText: userLogin,
+            displayName: userPersonalDetail.displayName,
+            icons: [userPersonalDetail.avatar],
+            keyForList: login,
```

Comment:
is there a reason to not use `userLogin` here as well?

Reply:
Not really, just thought using it for the key would be fine, but will do it for consistency's sake. Thank you for the suggestion.